app/src/main/kotlin/com/wire/android/ui/authentication/devices/remove/RemoveDeviceScreen.kt

```diff
@@ -130,15 +142,17 @@ private fun RemoveDeviceContent(
     val lazyListState = rememberLazyListState()
     Scaffold(topBar = {
         RemoveDeviceTopBar(
+            title = title,
+            description = description,
             elevation = lazyListState.rememberTopBarElevationState().value,
-            onBackButtonClicked = onBackButtonClicked
+            onBackButtonClicked = onBackButtonClicked,
+            navigationIconType = navigationIconType
         )
     }) { internalPadding ->
         Box(modifier = Modifier.padding(internalPadding)) {
             when (state.isLoadingClientsList) {
-                true -> RemoveDeviceItemsList(lazyListState, List(10) { Device() }, true, onItemClicked)
-                false -> RemoveDeviceItemsList(lazyListState, state.deviceList, false, onItemClicked)
-
+                true -> RemoveDeviceItemsList(lazyListState, List(5) { Device() }, true, onItemClicked, state.currentDevice)
+                false -> RemoveDeviceItemsList(lazyListState, state.deviceList, false, onItemClicked, state.currentDevice)
             }
```

Comment:
Wouldn't maybe be better just adding `List(5) { Device() }` and `state.deviceList` from an `if` instead of rewriting all the same object call twice? 🤔 

Something like this:
```
RemoveDeviceItemsList(
        lazyListState,
        if (state.isLoadingClientsList) List(5) { Device() } else state.deviceList,
        state.isLoadingClientsList,
        onItemClicked,
        state.currentDevice
)
```

Reply:
good idea!